components/modelregistry/modelregistry.go

```diff
@@ -79,6 +81,12 @@ func (m *ModelRegistry) ReconcileComponent(_ context.Context, cli client.Client,
 				return err
 			}
 		}
+
+		// Create odh-model-registries namespace
+		_, err := cluster.CreateNamespace(cli, "odh-model-registries", cluster.WithLabels(labels.ODH.OwnedNamespace, "true"))
+		if err != nil {
+			return err
+		}
```

Comment:
We don't have [unified labels in place yet](https://github.com/bartoszmajsak/opendatahub-architecture-decision-records/blob/5df775f582393dc640cd805080184a038d2b56f0/platform/ODH-ADR-0001-k8s-labels-standard.md#L84),

but I was thinking we can establish a bit more meaningful relationship here:

```suggestion
	modelRegistryLabels := cluster.WithLabels(
		labels.ODH.OwnedNamespace, "true",
		labels.K8SCommon.PartOf, ComponentName,
		labels.ODH.Component(ComponentName), "true",
	)
	_, err := cluster.CreateNamespace(cli, "odh-model-registries", modelRegistryLabels)
	if err != nil {
		return err
	}
```

`modelRegistryLabels` can also be a top-level var, as it's static. WDYT @zdtsw @dhirajsb?

Reply:
I wonder if we shouldn't add short comment on "why" we do not set ownership for this namespace, or more generally speaking, why the operator shouldn't remove it.

So explaining this decision from the PR description, but on the code level:

> this NS wont get deleted even MR component is set to Removed/Unmanagaed or ODH operator is being uninstalled.

Reply:
i am good to make modelRegistryLabels with labels, but want to discuss more regarding 2 extra/new labels

Reply:
did update with
- use variable to catch label
- add inline comments to keep namespace

wait for some offline discussion if should add 2 new labels or not